### PR TITLE
add example notebook and instructions on setup of jupyterlab

### PR DESCRIPTION
**Pull request by manuel-berning**

### CODE REVIEW

File: examples/train_and_evaluate_custom_model.ipynb [hunk 1]
```diff
@@ -0,0 +1,1996 @@
...
+      "            \"'min_samples_split': 20, 'min_weight_fraction_leaf': 0.0, \"\n",
+      "            \"'n_estimators': 64, 'n_jobs': 8, 'oob_score': False, 'verbose': \"\n",
+      "            \"0, 'warm_start': False}\\n\"\n",
+      "            '[2020-02-20 15:30:50 +0000] [8] [INFO] [classifier] 0.909 for '\n",
+      "            \"{'bootstrap': True, 'class_weight': None, 'criterion': 'gini', \"\n",
+      "            \"'max_depth': None, 'max_features': 'auto', 'max_leaf_nodes': \"\n",
+      "            \"None, 'min_impurity_decrease': 0.0, 'min_samples_leaf': 5, \"\n",
+      "            \"'min_samples_split': 20, 'min_weight_fraction_leaf': 0.0, \"\n",
+      "            \"'n_estimators': 128, 'n_jobs': 8, 'oob_score': False, 'verbose': \"\n",
+      "
```
Comment (lextiz) [inline examples/train_and_evaluate_custom_model.ipynb]: We might want to remove the printouts from the code and increase the library logging level.

**Comment by manuel-berning** [comment]
> [CLA assistant check](https://cla-assistant.io/SAP/document-classification-client?pullRequest=1)
> All committers have signed the CLA.

File: examples/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+In order to use the example `train_and_evaluate_custom_model.ipynb` example two prerequisites are:
+
+1. You have access to a service key for the SAP AI Busniess Services Document Classification service
+2. You need either a Jupyterhub instance (below are the instructions for setup either locally on your laptop in in SAP converged cloud instance)
```
[the remark this answers is not in the document]
Comment (manuel-berning) [reply examples/README.md]: removed. Only local setup was described below anyway. Thanks for noticing

File: examples/train_and_evaluate_custom_model.ipynb [hunk 3]
```diff
@@ -0,0 +1,1996 @@
...
+    "For the demo, we are using a Jupyter Notebook and make use of a special Python client library to invoke the most important functions of the DC REST API. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Dataset\n",
+    "\n",
+    "In this notebook we use a minimal dataset `ML-BDP-dataset-HifiManuals`, check it out by clicking the folder item at the top left of this page and navigating to the folder AFTER executing the next cell which clones this dataset from the SAP corporate github. Note that this is a quite small dataset and an easy classification task, try to create a more challenging one to make full use of the performance visualizations at the end of this notebook.\n",
```
[the remark this answers is not in the document]
Comment (manuel-berning) [reply examples/train_and_evaluate_custom_model.ipynb]: As discussed in slack I think this is main open topic and it should be clarified first which dataset we can want to use here.

File: examples/README.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+### Service key
+
+You will need to have access to SAP Cloud Plattform given via the [SAP Cloud Platform Enterprise Agreement](https://wiki.scn.sap.com/wiki/pages/viewpage.action?pageId=489198716)
+
+In order to get a service key please have a look at this [SAP Help Portal article](https://help.sap.com/viewer/65de2977205c403bbc107264b8eccf4b/Cloud/en-US/4514a14ab6424d9f84f1b8650df609ce.html)
+
+### Example how to set up local Jupyterlab instance
```
[the remark this answers is not in the document]
Comment (manuel-berning) [reply examples/README.md]: Changed.

File: examples/train_and_evaluate_custom_model.ipynb (hunk 3 above)
[the remark this answers is not in the document]
Comment (manuel-berning) [reply examples/train_and_evaluate_custom_model.ipynb]: changed

File: examples/train_and_evaluate_custom_model.ipynb [hunk 5]
```diff
@@ -0,0 +1,1996 @@
...
+    "\n",
+    "NOTE: This notebook currently covers multi-characteristic multi-class classification. We are working on adding support for multi-label classification as well.\n",
```
[the remark this answers is not in the document]
Comment (manuel-berning) [reply examples/train_and_evaluate_custom_model.ipynb]: done https://github.com/SAP/document-classification-client/issues/2

File: examples/train_and_evaluate_custom_model.ipynb [hunk 6]
```diff
@@ -0,0 +1,1996 @@
...
+    "The Document Classification Service Python Client accepts datasets as a folder with a pair of files ending in .pdf and .json for each document to be used for training. The .pdf file is the original document while the .json file specifies the ground truth annotation. You can open the .json by double clicking to check out the expected format of the ground truth specification"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%cd /home/jovyan/work\n",
```
[the remark this answers is not in the document]
Comment (manuel-berning) [reply examples/train_and_evaluate_custom_model.ipynb]: Removed

File: examples/train_and_evaluate_custom_model.ipynb (hunk 1 above)
Comment (manuel-berning) [reply examples/train_and_evaluate_custom_model.ipynb]: Done

**Pull request merged**